docs/api/js/providers.md

```diff
@@ -69,7 +69,7 @@ import { Provider } from "zksync-web3";
 
 const provider = new Provider("https://testnet.era.zksync.dev");
 
-const USDC_L2_ADDRESS = "0x0faF6df7054946141266420b43783387A78d82A9";
+const USDC_L2_ADDRESS = "<YOUR_USDC_L2_ADDRESS>";
 // Getting  USDC balance of account 0x0614BB23D91625E60c24AAD6a2E6e2c03461ebC5 at the latest processed block
 console.log(await provider.getBalance("0x0614BB23D91625E60c24AAD6a2E6e2c03461ebC5", "latest", USDC_L2_ADDRESS));
```

Comment:
Please double check, I see here we have an issue as everywhere now... https://zksync2-testnet.zkscan.io/tokens 0x0faF6df7054946141266420b43783387A78d82A9 - is a common token, if we are going to remove it everywhere, we should mention the link where to find this address: https://zksync2-testnet.zkscan.io/tokens. 
![image](https://github.com/matter-labs/zksync-web-era-docs/assets/51121511/266ae558-973d-4b7d-93c5-c0b20508805d)
but 0x0614BB23D91625E60c24AAD6a2E6e2c03461ebC5 is a private address, we can delete it and comment that user should use its own here.

Reply:
@pcheremu I've updated it.